bin/build-sprite-function.js

```diff
@@ -0,0 +1,25 @@
+import defaultAttrs from '../src/default-attrs.json';
+
+/**
+ * Renders a SVG symbol tag
+ * @param {*} name The name of the icon
+ * @param {*} contents The contents of the icon
```

Comment:
Could you replace `*` with the actual type? In this case I think both of them should be `string`. Also it looks like this is missing an `@returns` statement